docs/mine-hnt/validators/requirements.mdx

```diff
@@ -2,29 +2,34 @@
 id: requirements
 title: Validator Requirements
 hide_title: true
-sidebar_label: Technical Requirements
+sidebar_label: Validator Requirements
 slug: /mine-hnt/validators/requirements
 ---
 
 import useBaseUrl from "@docusaurus/useBaseUrl";
 
+# Staking Requirements
+
+10,000 HNT is required to stake a single validator
+
+
 # Technical Requirements
 
-Based on early tests, validator node requirements include:
+Mainnet validator system requirements:
 
 - Operating system: 
   - Planning on running from docker: Any linux based system capable of running docker
   - Planning on running from source: Ubuntu 20.04 LTS
 - CPU: 
-  - Intel, AMD or ARM based processor 3.0 GHz or above with 2 cores or equivalent
+  - Intel, AMD or ARM based processor 3.0 GHz or above with 2 or more cores
```

Comment:
IIRC, the 3.0Ghz wasn't actually needed? I think many of us on EC2 or GCP or similar are getting 2.5Ghz cores or not much more?

Reply:
updated to 2.5Ghz or above.